src/components/LeaderBoard.tsx

```diff
@@ -0,0 +1,50 @@
+import { TriangleUpIcon } from "@chakra-ui/icons";
+import { Box, Card, CardBody, Heading, Stack } from "@chakra-ui/react";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { LeaderboardDog } from "../types/dog";
+
+export function Leaderboard(): JSX.Element {
+    const [leaderboard, setLeaderboard] = useState<LeaderboardDog[]>();
+
+    async function getAndSetLeaderboard() {
+        const result = await axios.get(
+            "https://dog-breed-voting-back-end.onrender.com/leaderboard"
+        );
+        setLeaderboard(result.data);
+    }
+
+    useEffect(() => {
+        getAndSetLeaderboard();
+    }, []);
+    console.log(leaderboard);
+    function formatLeaderboardDog(dog: LeaderboardDog) {
```

Comment:
for readability and best practices, suggestion would be to isolate component and give it a more descriptive name that shows it is a component.